client/tests/stubs.js

```diff
@@ -36,16 +38,13 @@ global.wagtailConfig = {
   ACTIVE_LOCALE: 'en',
 };
 
-const script = document.createElement('script');
-script.type = 'application/json';
-script.id = 'wagtail-config';
-script.textContent = JSON.stringify({
-  CSRF_HEADER_NAME: 'x-xsrf-token',
-  CSRF_TOKEN: 'potato',
+const configScript = Object.assign(document.createElement('script'), {
+  id: 'wagtail-config',
+  textContent: JSON.stringify(wagtailConfig),
+  type: 'application/json',
 });
-document.body.appendChild(script);
 
-global.wagtailVersion = '1.6a1';
```

Comment:
`window.wagtailVersion` is not used by any code, or any tests, so I have removed it.